bin/tictactoe

```diff
@@ -1,3 +1,156 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+puts
```

Comment:
Thanks for this suggestion. I've created the ```lib/tic_tac_toe_cli.rb``` file and placed ```call``` in there along with the other methods relating to the CLI logic, which ```call``` calls. Is that an acceptable way of implementing it?